draft-ietf-ppm-dap.md

```diff
@@ -2121,6 +2128,54 @@ send an HTTP DELETE request to
 `{helper}/tasks/{task-id}/aggregation_jobs/{aggregation-job-id}` so that the
 Helper knows it can clean up its state.
 
+#### Batch Buckets {#batch-buckets}
+
+When aggregation recovers an output share, it must be stored into an appropriate
+"batch bucket", which is defined in this section. The data stored in a batch
+bucket is kept for eventual use in the {{collect-flow}}.
+
+Batch buckets are indexed by an aggregation parameter, as well as a batch
```

Comment:
Since we no longer support multicollection, perhaps it makes sense to remove the aggregation parameter from the index.

Reply:
Good point!

As written, we do still allow aggregators to aggregate over an arbitrary number of aggregation parameters; the requirement is that we only allow collection of a batch over a single aggregation parameter.

I removed the index & added a few sentences requiring the Leader & Helper to validate that the aggregation parameter in the `CollectionJobReq` & `AggregateShareReq` match what was used for aggregation of the relevant batch.

But this might still be tricky: a naive Helper that permits aggregation of a batch with multiple aggregation parameters, and doesn't account for there being multiple aggregation parameters in its batch bucket implementation, might aggregate into batch buckets "cross-aggregation-parameter." We might want to block this, but I'm not sure this PR is the right place to do so. Or maybe an implementation note is sufficient?